modules/search/instant-search/components/search-results.scss

```diff
@@ -121,3 +129,18 @@
 		font-size: 0.9em;
 	}
 }
+
+.jetpack-instant-search__overlay-close {
+    background: none;
+    border: none;
+    display: inline-block;
+    margin: 0;
+    text-decoration: none;
+    cursor: pointer;
+    -webkit-appearance: none;
+    -moz-appearance: none;
```

Comment:
Could we use [`appearance: none;`](https://developer.mozilla.org/en-US/docs/Web/CSS/appearance) here? The build process will add these vendor prefixes as necessary.